Packs/MailListener/ReleaseNotes/1_0_52.md

```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### Mail Listener v2
+- Updated the Docker image to: *demisto/py3-tools:1.0.0.87415*.
+
+- Fixed an issue where fetching emails from an empty folder returned an error.
```

Comment:
```suggestion
- Fixed an issue where fetching emails from an empty folder returned an error.
- Fixed an issue where we first email in a folder was skipped.
```

Reply:
Not exactly, the firs email issue was alredy merged in previos PR, but now that the custemur confirmed it solves the duplicate issue, I will add that.